### ISSUE TITLE
File history scrolling does not work correctly for some files

### ISSUE THREAD

**Issue by vmakarenko** [MPDL/KEEPER#349]
Observed on the shared encrypted library managed by MPDL:
A file has 644 changes in full history, but only last 207 can reached from File History interface.
Full history is only accessible by patching of `seahub/seahub/utils/file_revisions.py`: 
```python
def get_file_revisions_within_limit(repo_id, path, commit_id=None, limit=50):
    # add string here
    limit=-1                                                             
...
```
Note: The behavior should be same for WebAPI, because WebAPI uses same backend method as web interface.

**Comment by MPDLRanger** [comment]
recheck with 7.1.3

**Comment by iosifpeterfi** [comment]
Fixed, check details here: https://download.seafile.com/published/support/server-upgrade/pro-7.0.md

Check the related commit here: https://github.com/MPDL/KEEPER/commit/2609537a27849575cc8040f488a3ae92c2bdc63c

**Comment by vmakarenko** [comment]
fixed in 2.1_hotfixes: https://github.com/MPDL/KEEPER/commit/2609537a27849575cc8040f488a3ae92c2bdc63c
